juniper_official/chassis/check-psm-power-usage-state.rule

```diff
@@ -91,41 +98,58 @@ healthbot {
                 description "Sets health based on increasing psm power usage";
                 frequency 1offset;
                 /*
-                 * Sets color to red when psm-power-usage is greater than psm-power-usage-threshold
+                 * Sets color to green when psm-power-usage is lesser than psm-power-usage-low-threshold
                  * 
                  */				
-                term is-power-usage-abnormal {
+                term power-usage-normal {
                     when {
-                        greater-than-or-equal-to "$psm-power-usage" "$psm-power-usage-threshold";
+                        less-than "$psm-power-usage" "$psm-power-usage-low-threshold";
+                        does-not-match-with "$psm-state" offline {
+                            ignore-case;
+                        }
                     }
                     then {
                         status {
-                            color red;
-                            message "$psm power usage($psm-power-usage) exceeds threshold";
+                            color green;
+                            message "$psm power usage($psm-power-usage) is normal";
                         }
                     }
                 }
                 /*
-                 * Sets color to green when psm-power-usage is normal.
+                 * Sets color to yellow when psm-power-usage is greater than psm-power-usage-low-threshold
                  * 
                  */				
-                term power-usage-normal {
+                term is-power-usage-exceeds-low-threshold {
                     when {
-                        does-not-match-with "$psm-state" offline {
-                            ignore-case;
+                        greater-than-or-equal-to "$psm-power-usage" "$psm-power-usage-low-threshold";
+                        less-than "$psm-power-usage" "$psm-power-usage-high-threshold";
+                    }
+                    then {
+                        status {
+                            color yellow;
+                            message "$psm power usage($psm-power-usage) exceeds low threshold (50)";
```

Comment:
Try to follow similarity/format of thresholds in all rule's of all trigger messages. I see dynamic values in some rules and static in some rule's messages. 

e.g. 
Static:
message "$psm power usage($psm-power-usage) exceeds low threshold (50)";
Dynamic:
message "$fan-name RPM percent $rpm-percent($measurement RPM) is greater than $rpm-high-threshold"

Reply:
Changed to dynamic value.